schema/me/conversation/index.js

```diff
@@ -208,27 +232,26 @@ export const ConversationFields = {
     resolve: ({ id, from_email }, options, req, { rootValue: { accessToken } }) => {
       const { page, size, offset } = parseRelayOptions(options)
       const impulseParams = { page, size, conversation_id: id, "expand[]": "deliveries" }
-      return gravity
-        .with(accessToken, { method: "POST" })("me/token", {
-          client_application_id: IMPULSE_APPLICATION_ID,
-        })
-        .then(data => {
-          return impulse
-            .with(data.token, { method: "GET" })(`message_details`, impulseParams)
-            .then(({ total_count, message_details }) => {
-              // Inject the convesation initiator's email into each message payload
-              // so we can tell if the user sent a particular message.
-              /* eslint-disable no-param-reassign */
-              message_details = message_details.map(message => {
-                return merge(message, { conversation_from_address: from_email })
-              })
-              /* eslint-disable no-param-reassign */
-              return connectionFromArraySlice(message_details, options, {
-                arrayLength: total_count,
-                sliceStart: offset,
-              })
-            })
+      return gravity.with(accessToken, { method: "POST" })("me/token", {
+        client_application_id: IMPULSE_APPLICATION_ID,
+      }).then(data => {
+        return impulse.with(data.token, { method: "GET" })(
+          `message_details`,
+          impulseParams
+        ).then(({ total_count, message_details }) => {
+          // Inject the convesation initiator's email into each message payload
+          // so we can tell if the user sent a particular message.
+          /* eslint-disable no-param-reassign */
+          message_details = message_details.map(message => {
+            return merge(message, { conversation_from_address: from_email })
+          })
+          /* eslint-disable no-param-reassign */
+          return connectionFromArraySlice(message_details, options, {
+            arrayLength: total_count,
+            sliceStart: offset,
+          })
```

Comment:
These were just automatic changes from the pre-commit linter hook, this stuff is all tested.